docs/de/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# Middleware
+
+Sie können Middleware zu **FastAPI**-Anwendungen hinzufügen.
+
+Eine "Middleware" ist eine Funktion, die mit jedem **Request** arbeitet, bevor er von einer bestimmten *Pfadoperation* verarbeitet wird. Und auch mit jeder **Response**, bevor sie zurückgegeben wird.
```

Comment:
Schlage `„Middleware“` vor (typografische Anführungszeichen). Hab das gerade in allen meinen PRs geändert.